.github/actions/prepare-workspace/action.yml

```diff
@@ -5,6 +5,11 @@ inputs:
     description: Cache infix used in cache actions
     required: false
     default: ${{ github.workflow }}
+    
+  go_version:
+    description: Golang version for cache keys
+    required: false
+    default: go1.19.2
```

Comment:
So now we have to keep this in sync with what's in the Makefile?

Reply:
I wonder if we can just determine Go version in the workflow job using `go version` and store it in the variable. @gzigzigzeo What do you think, would that work? Then we wouldn't need to define any versions here or come up with a way to get it from the Makefile.

Reply:
@r0mant Yes, we could, but this definition would be implicit. There's no way to call `go version` for default value here. I've implemented it as like as go cache paths.